src/main/java/com/Kh033Java/travelbook/controller/CountryController.java

```diff
@@ -0,0 +1,53 @@
+package com.Kh033Java.travelbook.controller;
+
+
+import com.Kh033Java.travelbook.entity.Note;
+import com.Kh033Java.travelbook.entity.Photo;
+import com.Kh033Java.travelbook.entity.Plan;
+import com.Kh033Java.travelbook.service.CountryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping("/api/v1/country")
+public class CountryController {
```

Comment:
methods in this controller doesnt work (return nothing). delete incoming relationships in entities and rewrite CountryService